Some tests do #include  relative to the top-level, so add top_srcdir to the
include path as otherwise out-of-tree builds can't find the files.

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>
---
 test/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/Makefile.am b/test/Makefile.am
index 2b504bd..6688b41 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -1,7 +1,7 @@
 noinst_PROGRAMS = mtdev-mapgen mtdev-matching mtdev-kernel
 bin_PROGRAMS = mtdev-test
 
-AM_CPPFLAGS=-I$(top_srcdir)/include/
+AM_CPPFLAGS=-I$(top_srcdir) -I$(top_srcdir)/include/
 
 mtdev_mapgen_SOURCES = mtdev-mapgen.c
 
-- 
1.7.10.4
